### PR TITLE
fix: id:50283 remove extra code from scss  for divider styling

### PR DESCRIPTION
**Pull request by Farhan-emgage**
remove extra code from scss  for divider styling

### CODE REVIEW

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/50283

File: src/components/ComboBox/ComboBox.tsx [hunk 1]
```diff
@@ -86,13 +86,13 @@ class ComboBox extends React.PureComponent<Props, State> {
...
-  
+  }
+
   componentDidMount() {
     const { items } = this.props;
     if (items && items[0]) {
-      const { type = "" } = items && items[0];
-      if (type !== "Tabuler") {
+      const { type = '' } = items && items[0];
+      if (type !== 'Tabuler') {
```
Comment (weegrom) [inline src/components/ComboBox/ComboBox.tsx:95]: @Farhan-emgage I know this is not related to this issue, but can you please correct the spelling if this itemType option. The correct spelling is "Tabular". You'll also need to check our other repo's to see if ComboBox is used with itemType set to "Tabuler" and change it to "Tabular". Thanks!

**Review by weegrom — COMMENTED** [review]
Apart from the miss-spelling this looks good.

File: src/components/ComboBox/ComboBox.tsx (hunk 1 above)
Comment (Farhan-emgage) [reply src/components/ComboBox/ComboBox.tsx:95]: @weegrom Ok i will check

Comment (weegrom) [reply src/components/ComboBox/ComboBox.tsx:95]: @Farhan-emgage please provide an update as to when this request will be completed.

**Pull request closed without merging**